### PR TITLE
Issue#1920 Replaced instances of PROJECT_NAME in cmake files

### PR DESCRIPTION
**Pull request by rudra2112**
Replaced instances of ${PROJECT_NAME} with project names in cmake files.

### CODE REVIEW

File: moveit_core/robot_state/CMakeLists.txt [hunk 1]
```diff
@@ -8,7 +8,7 @@ target_include_directories(moveit_robot_state PUBLIC
   $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>
   $<INSTALL_INTERFACE:include/moveit_core>
 )
-set_target_properties(moveit_robot_state PROPERTIES VERSION ${${PROJECT_NAME}_VERSION})
+set_target_properties(moveit_robot_state PROPERTIES VERSION "${${PROJECT_NAME}_VERSION}")
```
Comment (Abishalini) [inline moveit_core/robot_state/CMakeLists.txt:11]: Replace `PROJECT_NAME` here

File: moveit_core/robot_trajectory/CMakeLists.txt [hunk 2]
```diff
@@ -3,7 +3,7 @@ target_include_directories(moveit_robot_trajectory PUBLIC
   $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>
   $<INSTALL_INTERFACE:include/moveit_core>
 )
-set_target_properties(moveit_robot_trajectory PROPERTIES VERSION ${${PROJECT_NAME}_VERSION})
+set_target_properties(moveit_robot_trajectory PROPERTIES VERSION "${${PROJECT_NAME}_VERSION}")
```
Comment (Abishalini) [inline moveit_core/robot_trajectory/CMakeLists.txt:6]: Replace PROJECT_NAME here

File: moveit_common/CMakeLists.txt [hunk 3]
```diff
@@ -14,5 +14,5 @@ ament_package(
 
 install(
   DIRECTORY cmake
-  DESTINATION share/${PROJECT_NAME}
+  DESTINATION share/moveit_common NONE
```
Comment (Abishalini) [inline moveit_common/CMakeLists.txt:17]:
```suggestion
  DESTINATION share/moveit_common
```

**Comment by Abishalini** [comment]
Here is more information about code style guidelines - https://moveit.ros.org/documentation/contributing/code/
Most importantly, please set up pre-commit which will help pass the formatting CI check

File: moveit_planners/chomp/chomp_interface/CMakeLists.txt [hunk 4]
```diff
@@ -40,13 +40,13 @@ target_link_libraries(moveit_chomp_planner_plugin
 
 install(
   TARGETS moveit_chomp_interface moveit_chomp_planner_plugin
-  EXPORT ${PROJECT_NAME}Targets
+  EXPORT moveit_planners_chomp Targets
```
Comment (tylerjw) [inline moveit_planners/chomp/chomp_interface/CMakeLists.txt:43]: A space between the project name and Targets won't work.

File: moveit_setup_assistant/moveit_setup_assistant/CMakeLists.txt [hunk 5]
```diff
@@ -57,39 +57,39 @@ target_link_libraries(moveit_setup_assistant
...
   PROPERTIES OUTPUT_NAME collisions_updater
   PREFIX ""
 )
 
 install(
   TARGETS
     moveit_setup_assistant
-    ${PROJECT_NAME}_updater
-  EXPORT ${PROJECT_NAME}Targets
+    moveit_setup_assistant _updater
```
Comment (Abishalini) [inline moveit_setup_assistant/moveit_setup_assistant/CMakeLists.txt:79]:
```suggestion
    moveit_setup_assistant_updater
```

**Comment by rudra2112** [comment]
@tylerjw @Abishalini can you help me fix this conflict, I can't understand the issue.

**Pull request closed without merging**